db.go

```diff
@@ -40,7 +40,7 @@ func (p Postgres) MigrationLogDeleteSql() string {
 type Mysql struct{}
 
 func (m Mysql) SelectMigrationTableSql() string {
-	return "SELECT table_name FROM information_schema.tables WHERE table_name = ?"
+	return "SELECT table_name FROM information_schema.tables WHERE table_name = ? AND table_schema = ?"
```

Comment:
Though the connection points to a given database, the information_schema table itself is global, and not tied to the given db.

From the page you linked above:

> For a query that is limited to a specific constant table name, checks need be made only for the named table within the corresponding database directory. Example:

> SELECT TABLE_NAME FROM INFORMATION_SCHEMA.TABLES
> WHERE TABLE_SCHEMA = 'test' AND TABLE_NAME = 't1';